### ISSUE TITLE
Documentation of deploy

### ISSUE THREAD

**Issue by hugobarthelemy** [codegram/decidim-deploy-heroku#3]
Hello !

I tried to deploy my code on Heroku but it did not work.

The gem must be added in the Gemfile from the root of the general application or in the Gemfile developement app ?

It must be installed before the `` bundle exec rake development_app `` `?

One should push on Heroku the global application or only the development_app?

Thank you

**Comment by mrcasals** [comment]
Hi @hugobarthelemy!

You need to generate your definitive app with this guide:

https://github.com/decidim/decidim/blob/master/docs/getting_started.md#step-by-step

Once that's done you need to add this gem to the new Rails app:

https://github.com/codegram/decidim-deploy-heroku#usage

I'm closing this issue, if you need more help I'll reopen it :)

**Comment by mrcasals** [comment]
@aubryprieur The warning about bundler is just a warning, you need to upgrade your bundler gem.

What version of Ruby are you using?

**Comment by aubryprieur** [comment]
ruby 2.4.2p198

**Comment by jluiggi** [comment]
Same issue here when pushing to heroku...

```
  #  -------------------------------------------
remote:        Bundler Output: Fetching gem metadata from https://rubygems.org/.........
remote:        Fetching https://github.com/codegram/decidim-deploy-heroku.git
remote:        
remote:        [!] There was an error while loading `decidim-deploy-heroku.gemspec`: cannot load such file -- rails/generators
remote:        Does it try to require a relative path? That's been removed in Ruby 1.9. Bundler cannot continue.
remote:        
remote:         #  from /tmp/build_13ab3ff6935fbdb1152aa97c6974a76f/vendor/bundle/ruby/2.6.0/bundler/gems/decidim-deploy-heroku-96fb6d89b70f/decidim-deploy-heroku.gemspec:5

```

**Comment by jfoclpf** [comment]
I already added to GemFile

```
$ cat Gemfile
# frozen_string_literal: true

source "https://rubygems.org"

ruby RUBY_VERSION

gem "decidim", "0.21.0"
# gem "decidim-consultations", "0.21.0"
# gem "decidim-initiatives", "0.21.0"

gem "bootsnap", "~> 1.3"

gem "puma", "~> 4.3.3"
gem "uglifier", "~> 4.1"

gem "faker", "~> 1.9"

gem "figaro"

gem "decidim-deploy-heroku"

group :development, :test do
  gem "byebug", "~> 11.0", platform: :mri

  gem "decidim-dev", "0.21.0"
end

group :development do
  gem "letter_opener_web", "~> 1.3"
  gem "listen", "~> 3.1"
  gem "spring", "~> 2.0"
  gem "spring-watcher-listen", "~> 2.0"
  gem "web-console", "~> 3.5"
end

group :production do
  gem "passenger"
  gem 'delayed_job_active_record'
  gem "daemons"
end
```

And still I cannot even install the package
```
gem "decidim-deploy-heroku", git: "https://github.com/codegram/decidim-deploy-heroku.git"
ERROR:  While executing gem ... (Gem::CommandLineError)
    Unknown command decidim-deploy-heroku,
```

Can you provide more information?

**Comment by mrcasals** [comment]
I know I'm super late here, but this gem is not supposed to be deployed to production. I'll update the docs to reflect that this gem should only be used in `:development` group, and can be removed once used.

**Comment by mrcasals** [comment]
Closed by 2684552
